src/map/script.cpp

```diff
@@ -11925,6 +11924,19 @@ BUILDIN_FUNC(enablenpc)
 		return SCRIPT_CMD_FAILURE;
 	}
 
+	int char_id = 0;
+
+	if (script_hasdata(st, 3))
+		char_id = script_getnum(st, 3);
+	else if (!strcmp(command,"cloakoffnpcself") || !strcmp(command,"cloakonnpcself")) {
+		map_session_data* sd;
+
+		if (!script_rid2sd(sd))
+			return SCRIPT_CMD_SUCCESS;
```

Comment:
```suggestion
			return SCRIPT_CMD_FAILURE;
```
Shouldn't we technically return a failure here?

Reply:
`script_rid2sd` already reports the error displayed by `SCRIPT_CMD_FAILURE`, so `SCRIPT_CMD_SUCCESS` is the right value to return
https://github.com/rathena/rathena/blob/cfa9776a281a5e9431b3e4880a1300c0a7ac0100/src/map/script.cpp#L2672
https://github.com/rathena/rathena/blob/cfa9776a281a5e9431b3e4880a1300c0a7ac0100/src/map/script.cpp#L4134